frontend.sh

```diff
@@ -63,19 +63,50 @@ install() {
   fi
 }
 
-
 # Run tasks to build the project for distribution.
 build() {
   echo "Building project…"
   yarn run gulp build
 }
 
+# Fake our font files, if necessary, in public CI
+fake_fonts() {
+  echo "Faking font files if necessary…"
+  # We want to test Django collectstatic, but we might not have our webfont 
+  # files if this script is running somewhere public. Because cfgov-refresh
+  # uses the Django ManifestStaticFilesStorage backend, the collectstatic 
+  # command will fail if any referenced files are missing.
+  #
+  # Builds generated with webfont files place them in a static.in/0/fonts
```

Comment:
In the deployable zipfile case, I used `static.in/0/fonts` because our artifacts happen to use numbered subfolders of `static.in`. [Our documentation recommends](https://cfpb.github.io/cfgov-refresh/installation/#webfonts) putting the webfonts locally under `static.in/cfgov-fonts/fonts`. Would it make sense to use that subfolder instead?

Then, this script could check if the folder exists, and do nothing if so (which would be consistent with the "if necessary" in the logging statement above).

I guess one downside of doing this is that it might confuse people if they run this code to generate fake fonts and later wonder why the fonts don't look correct. So maybe it'd be better to more explicitly put the fake files under `static.in/fake-fonts/fonts` -- but still only do that if the real fonts don't exist in a sibling directory.

Reply:
@chosak I've taken your suggestion to use `static.in/fake-fonts/fonts` only if `static.in/cfgov-fonts/fonts` does not exist.